spanner/opencensus/src/test/java/com/example/spanner/opencensus/QueryStatsMetricSampleIT.java

```diff
@@ -0,0 +1,128 @@
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Integration tests for Cloud Spanner OpenCensus Query Stats metric examples.
+ */
+@RunWith(JUnit4.class)
+public class QueryStatsMetricSampleIT {
+
+  // The instance needs to exist for tests to pass.
+  private static String instanceId = "default-instance";
+  private static String databaseId = formatForTest(
+      System.getProperty("spanner.sample.database", "mysample"));
+  private static DatabaseId dbId;
+  private static DatabaseAdminClient dbClient;
+  private static Spanner spanner;
+
+  @BeforeClass
+  public static void createTestDatabase() throws Exception {
+    Preconditions.checkState(instanceId != null, "No instance id set");
+    final SpannerOptions options =
+        SpannerOptions.newBuilder().setAutoThrottleAdministrativeRequests().build();
+    spanner = options.getService();
+    dbClient = spanner.getDatabaseAdminClient();
+    dbId = DatabaseId.of(options.getProjectId(), instanceId, databaseId);
+    dbClient.dropDatabase(dbId.getInstanceId().getInstance(), dbId.getDatabase());
+    dbClient
+        .createDatabase(
+            instanceId,
+            databaseId,
+            ImmutableList.of(
+                "CREATE TABLE Albums ("
+                    + "  SingerId        INT64 NOT NULL,"
+                    + "  AlbumId         INT64 NOT NULL,"
+                    + "  AlbumTitle      STRING(MAX),"
+                    + "  MarketingBudget INT64"
+                    + ") PRIMARY KEY (AlbumId)"))
+        .get();
+  }
+
+  @AfterClass
+  public static void dropTestDatabase() {
+    dbClient.dropDatabase(dbId.getInstanceId().getInstance(), dbId.getDatabase());
+    spanner.close();
+  }
+
+  @Before
+  public void insertTestData() {
+    final DatabaseClient client = spanner.getDatabaseClient(dbId);
+    client.write(Arrays.asList(
+        Mutation.newInsertBuilder("Albums")
+            .set("SingerId")
+            .to(1L)
+            .set("AlbumId")
+            .to(1L)
+            .set("AlbumTitle")
+            .to("Title 1")
+            .build()
+    ));
+  }
+
+  @After
+  public void removeTestData() {
+    final DatabaseClient client = spanner.getDatabaseClient(dbId);
+    client.write(Collections.singletonList(Mutation.delete("Albums", KeySet.all())));
+  }
+
+  @Test
+  public void testQueryStatsMetricSample() {
+    final DatabaseClient client = spanner.getDatabaseClient(dbId);
+    final String out = runExample(client);
+
+    assertThat(out).contains("1 1 Title 1");
+  }
+
+  private String runExample(DatabaseClient client) {
+    PrintStream stdOut = System.out;
+    ByteArrayOutputStream bout = new ByteArrayOutputStream();
+    PrintStream out = new PrintStream(bout);
+    System.setOut(out);
+    QueryStatsMetricSample.captureQueryStatsMetric(client);
+    System.setOut(stdOut);
+    return bout.toString();
```

Comment:
Same comment as in the other file, please move the setup and teardown steps to separate methods with the @Before and @After annotations. Example:

```
  @Before
  public void captureOutput() {
    bout = new ByteArrayOutputStream();
    System.setOut(new PrintStream(bout));
  }

  @After
  public void resetOutput() {
    System.setOut(originalOut);
    bout.reset();
  }
```

Reply:
Done